CONTRIBUTING.md

```diff
@@ -241,6 +241,6 @@ As Linguist is a production dependency for GitHub we have a couple of workflow r
 [languages]: /lib/linguist/languages.yml
 [licenses]: https://github.com/github/linguist/blob/9b1023ed5d308cb3363a882531dea1e272b59977/vendor/licenses/config.yml#L4-L15
 [samples]: /samples
-[search-example]: https://github.com/search?utf8=%E2%9C%93&q=extension%3Aboot+NOT+nothack&type=Code&ref=searchresults
+[search-example]: https://github.com/search?utf8=%E2%9C%93&q=path%3A*.boot+NOT+nothack&type=Code&ref=searchresults
```

Comment:
This should really match the search we use in the PR template:

https://github.com/github-linguist/linguist/blob/9d9a26d8ee352536c418847cc5386309c107e1bf/.github/PULL_REQUEST_TEMPLATE.md?plain=1#L14

```suggestion
[search-example]: https://github.com/search?type=code&q=NOT+is%3Afork+path%3A*.boot
```